components/RunScript/index.tsx

```diff
@@ -17,16 +21,21 @@ import {
 import Flex from "../Flex";
 import { useSnapshot } from "valtio";
 import Select from "../Select";
+import Text from "../Text";
 import { saveFile } from "../../state/actions/saveFile";
+import { getErrors, getTags } from "../../utils/comment-parser";
+import toast from "react-hot-toast";
 
-Handlebars.registerHelper(
-  "customize_input",
-  function (/* dynamic arguments */) {
-    return new Handlebars.SafeString(arguments[0]);
+const generateHtmlTemplate = (code: string, data?: Record<string, any>) => {
+  let processString: string | undefined;
+  const process = { env: { NODE_ENV: "production" } } as any;
+  if (data) {
+    Object.keys(data).forEach(key => {
+      process.env[key] = data[key];
+    });
   }
-);
+  processString = JSON.stringify(process);
```

Comment:
why is process stringified, only to be parsed again?

Reply:
Cuz we are generating html template, which is a string, so passing data object stringified there. It's parsing is then done in script scope (when script is run by user) to be made available to user as normal object.